core/modules/system/system.install

```diff
@@ -14,7 +14,129 @@ function system_requirements($phase) {
   // Ensure translations don't break during installation.
   $t = get_t();
 
-  // Report Backdrop version
+  // System metrics.
+  if ($phase == 'runtime') {
+    $metrics = array();
+
+    $free_output = shell_exec('free');
```

Comment:
Caution! On shared hosting `shell_exec` might be turned off for security reasons. In that case the install would break.

Reply:
...yup, should add some defensive coding there, since `free` is not available on Windows either.